### PR TITLE
Support krew release

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/cybozu-go/accurate/issues/24

Release kubectl-accurate plugin to krew-index.

## Review Points

* Build the binary to be released using [goreleaser](https://github.com/goreleaser/goreleaser) and archive it as `.tar.gz`
  * krew plugins must be packaged as .zip or .tar.gz archives
  * https://krew.sigs.k8s.io/docs/developer-guide/plugin-manifest/#specifying-plugin-download-options
* Use krew-release-bot to update krew-index
  * https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/

## Work to be done after merging

* Add tar.gz archive to v0.1.0 release
  * This will be done manually
  * `GORELEASER_CURRENT_TAG=v0.1.0 goreleaser release --skip-publish --skip-announce --skip-validate --rm-dist`
* Send a Pull Request to https://github.com/kubernetes-sigs/krew-index

## Testing

I've tested it in my own repository that I forked.

Release:
https://github.com/d-kuro/accurate/releases/tag/v0.1.1
https://github.com/d-kuro/accurate/runs/3809549806

Generate Krew manifests:

```console
$ docker run -v (pwd):/tmp/ rajatjindal/krew-release-bot:v0.0.40 krew-release-bot template --tag v0.1.1 --template-file /tmp/.krew.yaml
time="2021-10-06T00:40:29Z" level=info msg="getting sha256 for https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_darwin_amd64.tar.gz"
time="2021-10-06T00:40:30Z" level=info msg="downloaded file /tmp/768965160/1633480829"
time="2021-10-06T00:40:30Z" level=info msg="getting sha256 for https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_darwin_arm64.tar.gz"
time="2021-10-06T00:40:30Z" level=info msg="downloaded file /tmp/737136999/1633480830"
time="2021-10-06T00:40:30Z" level=info msg="getting sha256 for https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_linux_amd64.tar.gz"
time="2021-10-06T00:40:31Z" level=info msg="downloaded file /tmp/849511322/1633480830"
time="2021-10-06T00:40:31Z" level=info msg="getting sha256 for https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_linux_arm64.tar.gz"
time="2021-10-06T00:40:31Z" level=info msg="downloaded file /tmp/972828721/1633480831"
time="2021-10-06T00:40:31Z" level=info msg="getting sha256 for https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_windows_amd64.tar.gz"
time="2021-10-06T00:40:32Z" level=info msg="downloaded file /tmp/954888156/1633480831"
apiVersion: krew.googlecontainertools.github.com/v1alpha2
kind: Plugin
metadata:
  name: accurate
spec:
  version: v0.1.1
  homepage: https://github.com/cybozu-go/accurate
  platforms:
    - selector:
        matchLabels:
          os: darwin
          arch: amd64
      uri: https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_darwin_amd64.tar.gz
      sha256: 5cd5f58c63c9fcbdeaaec16be012c5b178f9788d1b1250d92f84df0c39dc80a9
      bin: kubectl-accurate
      files:
        - from: kubectl-accurate
          to: .
        - from: LICENSE
          to: .
    - selector:
        matchLabels:
          os: darwin
          arch: arm64
      uri: https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_darwin_arm64.tar.gz
      sha256: 2593fe9adaf50d404222b7a0ff2fb75c69421e379e1024d29ace1684061d59ae
      bin: kubectl-accurate
      files:
        - from: kubectl-accurate
          to: .
        - from: LICENSE
          to: .
    - selector:
        matchLabels:
          os: linux
          arch: amd64
      uri: https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_linux_amd64.tar.gz
      sha256: 6a76b327c6ea5bac270f7d386f8b6cb8a45090b042c15da9e309ee83f1bc8f30
      bin: kubectl-accurate
      files:
        - from: kubectl-accurate
          to: .
        - from: LICENSE
          to: .
    - selector:
        matchLabels:
          os: linux
          arch: arm64
      uri: https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_linux_arm64.tar.gz
      sha256: 31075c62d9d9074e37f1a012cfa35af533c3317b9698a506f5b24c15029877c4
      bin: kubectl-accurate
      files:
        - from: kubectl-accurate
          to: .
        - from: LICENSE
          to: .
    - selector:
        matchLabels:
          os: windows
          arch: amd64
      uri: https://github.com/d-kuro/accurate/releases/download/v0.1.1/accurate_v0.1.1_windows_amd64.tar.gz
      sha256: 52a62a4851f6e030f59518458d043e0fc40bcd9e45b1e1233ad51888bcf7a540
      bin: kubectl-accurate.exe
      files:
        - from: kubectl-accurate.exe
          to: .
        - from: LICENSE
          to: .
  shortDescription: Manage Accurate features
  description: |
    kubectl-accurate is a subcommand of kubectl to manage Accurate features.
    Read more documentation at: https://cybozu-go.github.io/accurate/kubectl-accurate.html
```

Install krew manifests:

```console
$ kubectl krew install --manifest=plugin.yaml
Installing plugin: accurate
Installed plugin: accurate
\
 | Use this plugin:
 | 	kubectl accurate
 | Documentation:
 | 	https://github.com/cybozu-go/accurate
/
```

### CODE REVIEW

**Review by zoetrope — DISMISSED** [review]

**Comment by d-kuro** [comment]
@zoetrope Did you accidentally close the Pull Request?

File: docs/install-plugin.md [hunk 1]
```diff
@@ -24,17 +37,24 @@ It is strongly recommended to install `kubectl-accurate` though Accurate can be
     $ ARCH=$(go env GOARCH)
     ```
 
-2. Download the binary and put it in a directory of your `PATH`.
+3. Set `VERSION` to the accurate version
+
+   See the accurate release page: https://github.com/cybozu-go/accurate/releases
```
[the remark this answers is not in the document]
Comment (d-kuro) [reply docs/install-plugin.md]: Sorry for the delay.
The following commit fixes it:
https://github.com/cybozu-go/accurate/pull/23/commits/00493fbb75a846a5576fc294d367b488b5743aef

File: .krew.yaml [hunk 2]
```diff
@@ -61,7 +61,9 @@ spec:
...
         - from: LICENSE
           to: .
-  shortDescription: Manage Accurate features
+  shortDescription: Accurate is a Kubernetes controller for multi-tenancy.
   description: |
     kubectl-accurate is a subcommand of kubectl to manage Accurate features.
-    Read more documentation at: https://cybozu-go.github.io/accurate/kubectl-accurate.html
+    Accurate is a Kubernetes controller for multi-tenancy.
+    It propagates resources between namespaces accurately and allows tenant users to create/delete sub-namespaces.
+    Read more documentation at: https://cybozu-go.github.io/accurate/index.html
```
Comment (d-kuro) [inline .krew.yaml]: In the pull request to add to moco's krew index, it was pointed out to me, so I made the description more detailed.
https://github.com/kubernetes-sigs/krew-index/pull/1716#discussion_r741192314

Comment (d-kuro) [reply .krew.yaml]: https://github.com/kubernetes-sigs/krew-index/pull/1721#pullrequestreview-797095195

**Comment by zoetrope** [comment]
@d-kuro 
I'm sorry.

**Review by zoetrope — APPROVED** [review]
LGTM

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**